### ISSUE TITLE
Plans for this page?

### ISSUE THREAD

**Issue by Gaelan** [AgoraNomic/blog#1]
What were you planning on putting here? I was thinking this would either be the current content of agoranomic.org or the wiki.

**Comment by nixnull** [comment]
I think this should be replaced with the wiki. I can remove the CNAME from the wiki and rename the repo so the link works as expected.

As for the current contens of agoranomic.org - some of it can be subsummed but much of it is out of date. If/when comex/omd resurfaces, we can discuss CNAME-ing these github.io pages to agoranomic.org subdomains.

**Comment by inquisitivecrystal** [comment]
Compromise solution: move this page to /blog, and the wiki to /wiki
(doesn't make all that much sense to have it as the main page either). Then
copy what's at agoranomic.org right now, and move it to the home page. That
way we can work on it, and when comex/omd gets back the whole site can be
moved to the other domain.

**Comment by PubliusScriboniusScholasticus** [comment]
@bookofportals I like this idea, make this /blog, make the wiki /wiki and make agoranomic.org this.

**Comment by PubliusScriboniusScholasticus** [comment]
@bookofportals @nichdel @Gaelan I have gone ahead and done this, but it should be easy to change back.

**Comment by nixnull** [comment]
Please stop making changes without consesus. I'm uninterested in having my own work over-ridden by whim, and *will* stop supporting the project if that's to be the case.

**Comment by inquisitivecrystal** [comment]
@nichdel: I agree. @PubliusScriboniusScholasticus, could you please try to wait for people to talk a bit longer before acting? I admire your initiative, but as nichdel says, we need to get a bit more consensus before doing stuff? Meanwhile, nothing has been destroyed, no one's work has been overwritten as far as I can tell, and we're all working together here (see [wikipedia WP:BRD](https://en.wikipedia.org/wiki/Wikipedia:BOLD,_revert,_discuss_cycle)). It would be great if everyone if everyone could step back and discuss, and please please _try_ to assume that everyone here is acting in good faith.

**Comment by PubliusScriboniusScholasticus** [comment]
I am very sorry and in the future will do. I am used to using GitHub for
fast-paced things where we act then ask questions. If you want, I can undo
it.

**Comment by nixnull** [comment]
I believe everyone is acting in good faith, but I also believe that Agora typically moves at a weekly pace and would like to carry that timescale over. The Grace Hopper method is fine for content changes and obvious improvements, but it's quite a bit different for things like removing a CNAME (which changes the address of the wiki) without seeking agreement first. If you had waited a day (today is the first day of my weekend so I was saving substantive comments until tomorrow) you'd have know I didn't want things done as they were, and trouble could have been avoided via patience.

As far as the structure of the site as a whole, I still think the wiki should replace the current front page. I'd be fine with the front page of the wiki being more similar to that page, but my thoughts are basically these:

-As a wiki, any player can update it so it doesn't stagnant like the current homepage has
-The homepage and the wiki are conceptually similar in that they're both *about* Agora without being Agora. That's why I think they should be combined, whereas report repos should stay separate.
-I honestly see no point whatsoever to maintaining a blog.

Of course if I see evidence that my opinion is the minority, I'll accede; I just want to see that first.

**Comment by PubliusScriboniusScholasticus** [comment]
I would support merging the wiki with the main page, but I would like the
wiki to have all of the same new information and I would like the main page
archived. As to the blog, I think it would be a good place for archiving
longer works.

**Comment by nixnull** [comment]
I'll see about writing a 'mock-up' front page that combines the current wiki front page and current agoranomic.org one as soon as possible. Right now I'm thinking:

-Keep & Update the "Welcome to Agora" and Why Agora" sections
-Replace the Historical section with a link to a wiki page, which itself links to historical documents on the wiki (I've already imported the Thesis archive and G already has the CFJ archive)
-Replace the "Thinking of playing" section with a link to the page with new plaer info (or move that page's content to the front page, either way)
-Replace the contents of "Other Websites" with the links to the various officer sub-sites.

**Comment by nixnull** [comment]
Also WRT a blog, it might be good to treat the Newspaper as the blog, it seems like roughly the same idea. Thoughts, @PubliusScriboniusScholasticus ?

**Comment by Gaelan** [comment]
@Nichdel there is a standard header in /Header you can use

**Comment by PubliusScriboniusScholasticus** [comment]
I like the idea of using the blog for the newspaper, but I also want to
leave it open for theses and essays.

**Comment by PubliusScriboniusScholasticus** [comment]
I think the main page should keep the new player info right there for ease
of use.

**Comment by inquisitivecrystal** [comment]
I would very much like to keep our current main page. It's working fine right now. It probably needs to be updated, but it doesn't seem to make much sense to have the wiki be the first thing that comes up when people get to the site.

**Comment by Gaelan** [comment]
@bookofportals How does agoranomic.github.io look right now?

**Comment by inquisitivecrystal** [comment]
@Gaelan Looks good to me. 👍

**Comment by PubliusScriboniusScholasticus** [comment]
Does anyone else have any issues with the current page beyond updating information?

**Comment by inquisitivecrystal** [comment]
I've added some stuff. What do you guys think?

**Comment by BenjaminSchool** [comment]
@bookofportals It looks good.
